sunkit_pyvista/plotter.py

```diff
@@ -59,6 +61,33 @@ def _coords_to_xyz(self, coords):
                                 coords.y.to_value(R_sun),
                                 coords.z.to_value(R_sun)))
 
+    def set_camera_coordinates(self, coord):
+        """
+        Sets the inital camera position of the rendered plot.
+
+        Parameters
+        ----------
+        coords : `astropy.coordinates.SkyCoord`
+            Coordinates of the camera.
+        """
+        camera_position = self._coords_to_xyz(coord)
+        pos = tuple(camera_position[0])
+        self.plotter.camera.position = pos
+
+    def rotate_camera(self, angle: u.deg = None):
```

Comment:
Will this need a u.quantity decorator to ensure the unit type?

Reply:
Oh yeah I suppose.